### ISSUE TITLE
Add salesforce to streaming part.

### ISSUE THREAD

**Issue by myrulezzz** [mage-ai/mage-ai#1879]
Currently using simple salesforce  python library with oauth2 to subscribe to push topics on salesforce and sending you records to a kafka topic.Instead of running the kafka consumer manually i would like to see a salesforce option in streaming other than the one in the integration, so o can have up and running and monitoring the kafka consumer from salesforce throw mage.Is that possible? Thanks in advance

**Comment by wangxiaoyou1993** [comment]
Yes, that's possible. Thanks for creating the feature request. Just to clarify, you want to use Salesforce as the streaming source, right? And when do you need this feature by? We can add it to our roadmap

**Comment by myrulezzz** [comment]
Hi @wangxiaoyou1993. Yes that's right.Whenever you have it ready i am happy.When are you planning to add it?

**Comment by wangxiaoyou1993** [comment]
@myrulezzz , we can add it in February.

**Comment by myrulezzz** [comment]
Perfect thanks.let me know when you have it

**Comment by thomaschung408** [comment]
Hi @myrulezzz! Just checking in. Is this a feature you're still interested in?

**Comment by myrulezzz** [comment]
Hi @thomaschung408  yes of cource. It would be great. To be able to use salesforce as streaming source to load data from a push topic or platform event in to mage and then to kafka

**Comment by thomaschung408** [comment]
Got it! We have it on our roadmap for mid-May release: https://airtable.com/shrJctaGt4tvU27uF/tbl5bIlFkk2KhCpL4
We'll keep you posted!
